### DISCUSSION TITLE
In combined harvesting mode, can the cutter be folded during unloading

### DISCUSSION THREAD

**Discussion by shimq520** [Courseplay/Courseplay_FS22#2219] [Ideas]
In combined harvesting mode, can the cutter be folded during unloading?

**Comment by Tensuko** [comment]
You mean for example the foldable maize headers?

**Reply by shimq520** [reply]
For example, some long foldable or retractable cutters include corn, wheat, beet and potato cutters. If the cutter folds up when unloading, the unloading vehicle will not easily get stuck on the cutter.

**Reply by shimq520** [reply]
I have looked at the code, but I don't know how to modify it or which method to call to implement this function. So please help me add this function.Thank you!

**Reply by Tensuko** [reply]
I would like to know where your conflicts with the headers are.
I guess they are modded?
Wich one for example?

**Reply by shimq520** [reply]
Yes,there are mods.Form example,HOLMER 16m cutter,Cressoni Cutter 23m v2.0,Crazy 18M 30M 42M Corn and Sunflower Cutters v2.0.

**Reply by Tensuko** [reply]
Sry but we wont add an option for this then.
Adding an option for unrealistic tools is not our intention, sry.

**Reply by shimq520** [reply]
Thank you! Can you tell me which method is responsible for folding and unfolding the cutter? I want to try it myself.

**Reply by Tensuko** [reply]
Search for foldImplement.

**Reply by shimq520** [reply]
Thank you. I'll try.

**Reply by shimq520** [reply]
No available methods containing fold_implement were found.
![Uploading IMG_20230103_115830.jpg…]()
